src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -2631,40 +2247,14 @@ void WriteTabularLifeCycleCostReport(EnergyPlusData &state)
         columnWidth = 14; // array assignment - same for all columns
         tableBody.allocate(1, 16);
         tableBody = "";
-        rowHead(1) = "Construction";
-        rowHead(2) = "Salvage";
-        rowHead(3) = "Other Capital";
-        rowHead(4) = "Energy";
-        rowHead(5) = "Water";
-        rowHead(6) = "Maintenance";
-        rowHead(7) = "Repair";
-        rowHead(8) = "Operation";
-        rowHead(9) = "Replacement";
-        rowHead(10) = "Minor Overhaul";
-        rowHead(11) = "Major Overhaul";
-        rowHead(12) = "Other Operational";
-        rowHead(13) = "Total Energy";
-        rowHead(14) = "Total Operation";
-        rowHead(15) = "Total Capital";
-        rowHead(16) = "Grand Total";
+        for (int CashFlowCostCategory = CostCategory::Maintenance; CashFlowCostCategory <= CostCategory::TotGrand; ++CashFlowCostCategory) {
```

Comment:
I see, so this is why the table diffs appear.  I think that is acceptable and these loops are much nicer than the manually listed out chunks of code.